paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/FormControllerModule.kt

```diff
@@ -43,6 +43,8 @@ internal object FormControllerModule {
             cbcEligibility = CardBrandChoiceEligibility.Ineligible,
             billingDetailsCollectionConfiguration = PaymentSheet.BillingDetailsCollectionConfiguration(),
             requiresMandate = false,
+            linkConfigurationCoordinator = null,
```

Comment:
Why can this be null?

Reply:
`FormController` I believe is the legacy way for handling form events. We don't use it in `PaymentSheet` anymore but it does share the same logic transforming logic as `PaymentSheet` does. It is still used by `AddressElement` which does not support `Link` signup so this is safe to set to null.

Reply:
Makes sense. Want to add a comment and throw in the callback?